fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,4 +1,4 @@
-Requires Mattermost Server v5.9+. Older servers may not be able to connect or have unexpected behavior.
+Requires Mattermost Server v5.19+. Older servers may not be able to connect or have unexpected behavior.
```

Comment:
@amyblais and this should update the Android store description

Reply:
I wonder if this will cause any confusion. Will think if there is a better process in getting it updated closer to 16th

Reply:
this won't update until you publish the app in the store.

Reply:
The reason this has to be done now is because this will be uploaded with the apk to the store (waiting to be published) in order to have this done after the 16th is to manually update the metadata in the store before publishing and then update this whenever we want. Not really sure is needed as this file is really buried and only used for building the app. I get that the README.md can be treated differently.